testsuite/features/build_validation/create_bootstrap_repositories/srv_create_bootstrap_repositories.feature

```diff
@@ -62,6 +62,14 @@ Feature: Create bootstrap repositories
   Scenario: Create the bootstrap repository for a SLE Micro 5.3 SSH minion
     When I create the bootstrap repository for "slemicro53_ssh_minion" on the server
 
+@slemicro54_minion
+  Scenario: Create the bootstrap repository for a SLE Micro 5.4 minion
+    When I create the bootstrap repository for "slemicro54_minion" on the server
+
+@slemicro54_ssh_minion
+  Scenario: Create the bootstrap repository for a SLE Micro 5.4 SSH minion
+    When I create the bootstrap repository for "slemicro54_ssh_minion" on the server
+
```

Comment:
In theory we use the same bootstrap repository for Salt and SSH Minions, as we are using the same repos, so... I'm not sure if we are duplicating actions here.
I see this also happen in other previous commits.

Reply:
Good catch! Since both minions use the same OS they indeed use the same bootstrap repositories. I will add another commit that removes this duplication for all SLE Micro minions. Thank you!

Reply:
Yes please, because it's often we run into blocked process here, let's remove the duplication, once for its product

Reply:
Fixed.